src/site/site.xml

```diff
@@ -23,7 +23,6 @@ under the License.
          xsi:schemaLocation="http://maven.apache.org/DECORATION/1.8.1 https://maven.apache.org/xsd/decoration-1.8.1.xsd">
 
   <bannerLeft>
-    <name>${project.name}</name>
```

Comment:
I hope that can be also done for old version of m-site-p

Reply:
Just tried, does work.